### PR TITLE
Bump file.

### PR DESCRIPTION
**Pull request by bernaferrari**

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -3,14 +3,14 @@
 # found in the LICENSE file.
 
 name: process_runner
-version: 4.1.3
+version: 4.1.4
```
Comment (jakemac53) [inline pubspec.yaml]: It looks like 4.1.3 was never published, so we can leave that version (also that matches the CHANGELOG)

File: pubspec.yaml [hunk 2]
```diff
@@ -3,14 +3,14 @@
...
 description: A process invocation astraction for Dart that manages a multiprocess queue.
 homepage: https://github.com/google/process_runner
 
 dependencies:
   args: ^2.3.0
   async: ^2.5.0
-  file: ^6.1.0
+  file: ^7.0.0
```
Comment (jakemac53) [inline pubspec.yaml]: I would also leave these as `>=6.1.0 <8.0.0` where possible to ease the transition, given no change is needed here it must work with both :).

**Comment by jakemac53** [comment]
@gspencergoog will have to do the actual review and landing here

File: pubspec.yaml (hunk 1 above)
Comment (bernaferrari) [reply pubspec.yaml]: Ok, done.

**Review by gspencergoog — APPROVED** [review]
32384589-a60f0e74-c078-11e7-9bc1-e5b5287aea9d

**Comment by bernaferrari** [comment]
This should pass now.

**Pull request merged**